.github/workflows/cd.yml

```diff
@@ -132,7 +139,7 @@ jobs:
         uses: richardsimko/update-tag@v1
         if: ${{ contains(steps.version-change.outputs.CHANGED, 'templates@') && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch' && github.event.inputs.preid == 'alpha') }}
         with:
-          tag_name: "templates@0.0.0-alpha"
+          tag_name: "templates-0.0.0-alpha"
```

Comment:
dev branch should not has any templates@ as tag, the tag will influence the main version management.